DobieStation/DobieStation.vcxproj

```diff
@@ -24,6 +28,15 @@
     <IntermediateDirectory>release\</IntermediateDirectory>
     <PrimaryOutput>DobieStation</PrimaryOutput>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release Optimised|x64'" Label="Configuration">
+    <PlatformToolset>v141</PlatformToolset>
+    <OutputDirectory>release\</OutputDirectory>
+    <ATLMinimizesCRunTimeLibraryUsage>false</ATLMinimizesCRunTimeLibraryUsage>
+    <CharacterSet>NotSet</CharacterSet>
+    <ConfigurationType>Application</ConfigurationType>
+    <IntermediateDirectory>release\</IntermediateDirectory>
+    <PrimaryOutput>DobieStation</PrimaryOutput>
+  </PropertyGroup>
```

Comment:
Actually potentially a nice idea would be to use "Release Optimized" for the release and rename current release to "Devel" and enable symbols (faster debug).

That way you don't miss appveyor and you don't have to worry about US/UK spelling ;)

Reply:
Maybe, we can do that later :)